net/strongswan/files/ipsec.init

```diff
@@ -245,17 +270,17 @@ config_remote() {
 
 	[ "$gateway" = "any" ] && remote_gateway="%any" || remote_gateway="$gateway"
 
-	[ -z "$local_identifier" ] && {
+	[ -z "$local_gateway" ] && {
 		local ipdest
 
 		[ "$remote_gateway" = "%any" ] && ipdest="1.1.1.1" || ipdest="$remote_gateway"
-		local_gateway=`ip route get $ipdest | awk -F"src" '/src/{gsub(/ /,"");print $2}'`
+		local_gateway=`ip -o route get $ipdest | awk '/ src / { gsub(/^.* src /,""); gsub(/ .*$/, ""); print $0}'`
```

Comment:
How about `awk '/ src / { print gensub(/.* src ([^ ]+).*/,"\\1","1") }'`?

Reply:
Just saw this comment, but ended up doing something very similar in PR #14758.